tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -129,6 +130,39 @@ export class TestWorkspaceUtil implements ITestWorkspaceUtil {
         await this.deleteWorkspaceByName(namespace, workspaceName);
     }
 
+    // stop all run workspaces in the namespace
+    public async stopAllRunProjects(namespace: string) {
```

Comment:
IMHO, `stopAllRunningWorkspaces()` is more descriptive, becuase the method is currently stopping workspaces, not projects.